reliability-engineering/pipelines/build-gdsre-docker.yml

```diff
@@ -0,0 +1,37 @@
+resources:
+- name: gdsre-aws-ruby
+  type: docker-image
+  source:
+    # FIXME: move to docker hub
+    repository: ((readonly_private_ecr_repo_url))
+    tag: gdsre-aws-ruby-latest
+
+- name: ruby-2.6.1
+  type: docker-image
+  source:
+    repository: ruby
+    tag: "2.6.1"
+
+- name: gdsre-aws-ruby-git
+  type: git
+  source:
+    uri: https://github.com/alphagov/verify-infrastructure.git
```

Comment:
should we move the Dockerfile into this repo? ... seems weird being over there given it's name and generic use

Reply:
yes we should, but one step at a time

Reply:
ie not in this PR

Reply:
actually I've convinced myself otherwise, it'll be easier to move now than when the pipeline is active